
Epoch: 1/20











  3200/144000: [>...............................] - ETA 10725.5s











  6528/144000: [=>..............................] - ETA 10472.0s










  9568/144000: [==>.............................] - ETA 10240.5s











 12832/144000: [==>.............................] - ETA 9991.9ss











 16096/144000: [===>............................] - ETA 9743.3ss










 19072/144000: [====>...........................] - ETA 9516.7ss











 22336/144000: [====>...........................] - ETA 9268.1ss












 25472/144000: [=====>..........................] - ETA 9029.3ss











 28768/144000: [======>.........................] - ETA 8778.3ss











 32000/144000: [=======>........................] - ETA 8532.1ss











 35232/144000: [=======>........................] - ETA 8286.0ss











 38432/144000: [========>.......................] - ETA 8042.3ss










 41440/144000: [=========>......................] - ETA 7813.2ss











 44704/144000: [=========>......................] - ETA 7564.6ss











 47936/144000: [==========>.....................] - ETA 7318.5ss








 50240/144000: [===========>....................] - ETA 7143.0ss
Traceback (most recent call last):
  File "main_fusion.py", line 136, in <module>
    run(cfg)
  File "main_fusion.py", line 108, in run
    amp         = cfg['TRAINING']['amp'])
  File "/home/inho/df_detection/experiments_OC/train_fusion.py", line 184, in fit
    train_metrics = train(model1, model2, trainloader, criterion1, criterion2, consistency_rate, optimizer, log_interval, device, amp)
  File "/home/inho/df_detection/experiments_OC/train_fusion.py", line 48, in train
    features1, _ = model1(inputs)
  File "/home/inho/anaconda3/envs/df_detection/lib/python3.7/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/inho/df_detection/experiments_OC/model/modified_xception.py", line 231, in forward
    x = self.forward_features(x)
  File "/home/inho/df_detection/experiments_OC/model/modified_xception.py", line 203, in forward_features
    x = self.block4(x)
  File "/home/inho/anaconda3/envs/df_detection/lib/python3.7/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/inho/df_detection/experiments_OC/model/modified_xception.py", line 84, in forward
    x = self.rep(inp)
  File "/home/inho/anaconda3/envs/df_detection/lib/python3.7/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/inho/anaconda3/envs/df_detection/lib/python3.7/site-packages/torch/nn/modules/container.py", line 204, in forward
    input = module(input)
  File "/home/inho/anaconda3/envs/df_detection/lib/python3.7/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/inho/df_detection/experiments_OC/model/modified_xception.py", line 48, in forward
    x = self.pointwise(x)
  File "/home/inho/anaconda3/envs/df_detection/lib/python3.7/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/inho/anaconda3/envs/df_detection/lib/python3.7/site-packages/torch/nn/modules/conv.py", line 463, in forward
    return self._conv_forward(input, self.weight, self.bias)
  File "/home/inho/anaconda3/envs/df_detection/lib/python3.7/site-packages/torch/nn/modules/conv.py", line 460, in _conv_forward
    self.padding, self.dilation, self.groups)
KeyboardInterrupt